### PR TITLE
Remove unused parameters and structs related to Scheme

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

File: core/src/scheme/scheme.rs [hunk 1]
```diff
@@ -35,11 +35,6 @@ pub struct Scheme {
     pub name: String,
     /// What engine are we using for this?
     pub engine: Arc<dyn ConsensusEngine>,
-    /// Name of the subdir inside the main data dir to use for chain data and settings.
-    pub data_dir: String,
-
-    /// Known nodes on the network in enode format.
-    pub nodes: Vec<String>,
```
Comment (junha1) [inline core/src/scheme/scheme.rs:42]: How are these used in CodeChain?

Comment (majecty) [reply core/src/scheme/scheme.rs:42]: They were not used in CodeChain either. It seems that we copied scheme file Ethereum.

Comment (junha1) [reply core/src/scheme/scheme.rs:42]: Ok.

**Review by junha1 — APPROVED** [review]

**Review by junha1 — COMMENTED** [review]
There is a clippy error

**Comment by majecty** [comment]
@junha1 Thanks, I fixed it.

**Review by junha1 — APPROVED** [review]

**Pull request merged**